tests/foreman/smoke/test_api_smoke.py

```diff
@@ -608,6 +611,19 @@
 }
 
 
+def _bz_1184170_affects_server():
```

Comment:
Maybe worth creating some decorator/helper to check for server version. But this can be a future improvement.